moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -739,63 +589,28 @@ const ProjectComponentEdit = ({
       }
       // check if selected component is represented by lines or points
       setDrawLines(
-        data.moped_proj_components[0].moped_components.line_representation
+        selectedProjectComponent.moped_components.line_representation
       );
     }
   }
 
   // Populate component description if available and state is empty
   if (
-    data &&
-    !!data?.moped_proj_components[0]?.description &&
+    selectedProjectComponent &&
+    !!selectedProjectComponent?.description &&
     componentDescription === null
   ) {
-    setComponentDescription(data?.moped_proj_components[0]?.description);
+    setComponentDescription(selectedProjectComponent?.description);
   }
 
-  /**
-   * We need to populate the features we will need in order to build the
-   * geojson collection that we can feed it to the map.
-   */
-  if (data && editFeatureComponents.length === 0) {
```

Comment:
was able to slash this because we're not dealing with the `moped_proj_features_components` table anymore